Plugins/Public/playercntl_plugin/PimpShip.cpp

```diff
@@ -192,58 +208,35 @@ namespace PimpShip
 		if (!set_bEnablePimpShip)
 			return false;
 
-		uint iLocationID = 0;
-		pub::Player::GetLocation(iClientID, iLocationID);
-		if (set_mapDealers.find(iLocationID)==set_mapDealers.end())
+		if (!mapInfo[iClientID].bInPimpDealer)
 		{
-			uint iBaseID = 0;
-			pub::Player::GetBase(iClientID, iBaseID);
-			if (set_mapDealers.find(iBaseID)==set_mapDealers.end())
-			{
-				mapInfo[iClientID].bInPimpDealer = false;
-				mapInfo[iClientID].mapCurrEquip.clear();
-				PrintUserCmdText(iClientID, L"ERR ship pimping facilities not available at this base.");
-				return false;
-			}
+			PrintUserCmdText(iClientID, L"ERR Pimpship facilities are not available here");
+			return true;
 		}
 
-		mapInfo[iClientID].mapCurrEquip.clear();
-		mapInfo[iClientID].bInPimpDealer = true;
-
 		PrintUserCmdText(iClientID, L"Available ship pimping commands:");
+		PrintUserCmdText(iClientID, L"This facility costs " + ToMoneyStr(set_iCost) + L" credits to use per one item.");
```

Comment:
TODO: did we used to charge per-transaction and it's now per-item? that could be problematic

Reply:
I was convinced previous version took credits for every execution of /setitem command.

Reply:
Decided we're going to divide the price by 10 in the config.